release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -1267,6 +1332,11 @@ In {product-title} 4.12, support for {op-system} functionality is deprecated for
 
 While these hardware models remain fully supported in {product-title} 4.12, Red Hat recommends that you use later hardware models.
 
+[id="ocp-4-12-ne-kuryr"]
+==== Kuryr support for clusters that run on {rh-openstack}
+
+In {product-title} 4.12, support for Kuryr on clusters that run on {rh-openstack} is deprecated. In an upcoming release, support will be removed.
```

Comment:
"In an upcoming release" suggests it can be 4.13 and we aim for removal to happen no earlier than 4.14.

Reply:
Should be addressed by https://github.com/openshift/openshift-docs/pull/54280/commits/0000456c104ec8b89042a487f75370979fa4a506

Reply:
Yup, it is.